packages/esm-patient-chart-app/src/workspace/workspace-window.component.tsx

```diff
@@ -48,6 +48,7 @@ const WorkspaceWindow: React.FC<RouteComponentProps<ContextWorkspaceParams>> = (
   };
 
   const workspacesToRender = useMemo(() => {
+    windowSize.size === 'hidden' && updateWindowSize('reopened');
```

Comment:
`useMemo` shouldn't have side effects. From [the docs](https://reactjs.org/docs/hooks-reference.html#usememo):

> Don’t do anything there that you wouldn’t normally do while rendering. For example, side effects belong in useEffect, not useMemo.

Also this doesn't seem semantically related to the idea of "getting which workspaces to render" anyway.